salt/modules/beacons.py

```diff
@@ -251,3 +251,93 @@ def disable(**kwargs):
             # Effectively a no-op, since we can't really return without an event system
             ret['comment'] = 'Event module not available. Beacons enable job failed.'
     return ret
+
+
+def enable_beacon(name, **kwargs):
+    '''
```

Comment:
Oh, beacons/__initi__.py was added in 2015.5, but not modules/beacons.py. Sorry about the confusion there!